### PR TITLE
[Backport maintenance/2.17.x] Avoid `consider-using-f-string` on modulos with brackets in template …

### PR DESCRIPTION
**Pull request by jacobtylerwalls**
Backport of #8720 to 2.17.x

### CODE REVIEW

File: tests/functional/c/consider/consider_using_f_string.txt [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+consider-using-f-string:113:8:113:19:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:114:8:114:19:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:115:8:115:26:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:116:8:116:15:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:117:8:117:24:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:118:8:118:19:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:119:8:119:19:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:120:8:120:19:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:121:8:121:22:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
+consider-using-f-string:122:8:122:19:assignment_bad:Formatting a regular string which could be an f-string:UNDEFINED
```
Comment (jacobtylerwalls) [inline tests/functional/c/consider/consider_using_f_string.txt:30]:
```suggestion
consider-using-f-string:46:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:47:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:48:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:49:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:50:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:51:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:52:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:53:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:54:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:55:10:None:None:print_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:81:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:82:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:83:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:84:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:85:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:86:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:87:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:88:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:89:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:90:4:None:None:statement_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:113:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:114:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:115:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:116:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:117:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:118:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:119:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:120:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:121:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
consider-using-f-string:122:8:None:None:assignment_bad:Formatting a regular string which could be a f-string:UNDEFINED
```

**Comment by jacobtylerwalls** [comment]
To fix the merge conflict I regenerated the results but possibly with the wrong version of astroid, so I'll just reupdate from the pytest output on CI.

**Comment by jacobtylerwalls** [comment]
Maybe the backport isn't worth it.

**Review by Pierre-Sassoulas — APPROVED** [review]
I think it did not work at first because you took the python 3.7 output. 3.8 was the only data used until very recently because it had more information  (i.E. until we dumped python 3.7 entirely and simplified the functional tests helper).

**Pull request merged**